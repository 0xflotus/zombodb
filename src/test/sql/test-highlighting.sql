select zdb.highlight(ctid, 'payload.commits.message') from events where events ==> 'payload.commits.message:*beer*' order by id limit 10;

set enable_indexscan to off;
set enable_bitmapscan to off;
select zdb.highlight(ctid, 'payload.commits.message') from events where events ==> 'payload.commits.message:*beer*' order by id limit 10;