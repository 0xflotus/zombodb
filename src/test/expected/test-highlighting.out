select zdb.highlight(ctid, 'payload.commits.message') from events where events ==> 'payload.commits.message:*beer*' order by id limit 10;
                                                                highlight                                                                
-----------------------------------------------------------------------------------------------------------------------------------------
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Formatting beer fields</em>"}
 {"<em>Hauler bugfix for hauler bugfix :beers:</em>"}
 {"<em>Dont get brewery_guid from beers_t when it's not needed</em>","<em>More useful beers GetAll and Get integration test cases</em>"}
 {"<em>Adjusting show view for beer</em>"}
 {"<em>Fixing search beer</em>"}
 {"<em>Refactoring beer search</em>"}
(10 rows)

set enable_indexscan to off;
set enable_bitmapscan to off;
select zdb.highlight(ctid, 'payload.commits.message') from events where events ==> 'payload.commits.message:*beer*' order by id limit 10;
                                                                highlight                                                                
-----------------------------------------------------------------------------------------------------------------------------------------
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Format tweaking, added beer</em>"}
 {"<em>Formatting beer fields</em>"}
 {"<em>Hauler bugfix for hauler bugfix :beers:</em>"}
 {"<em>Dont get brewery_guid from beers_t when it's not needed</em>","<em>More useful beers GetAll and Get integration test cases</em>"}
 {"<em>Adjusting show view for beer</em>"}
 {"<em>Fixing search beer</em>"}
 {"<em>Refactoring beer search</em>"}
(10 rows)

